### ISSUE TITLE
Facebook oEmbeds are not converted into facebook_post components

### ISSUE THREAD

**Issue by webdeveric** [alleyinteractive/apple-news#425]
Using a Facebook oEmbed in the post content results in a `quote` component inside a `container` instead of the expected `facebook_post` component.

Plugin version: from `master` branch
WP version: `4.8.2`

---

![image](https://user-images.githubusercontent.com/1823514/30666086-d4df8b20-9e18-11e7-952d-8324561cb090.png)

**Comment by kevinfodness** [comment]
Hi @webdeveric—

Can you please reply back with the information requested here: https://github.com/alleyinteractive/apple-news/wiki/Usage#getting-help

Thanks!

**Comment by webdeveric** [comment]
@kevinfodness 

- Post content:

  ```
  https://www.facebook.com/permalink.php?story_fbid=1479413555506722&id=350849491696473
  ```
- `article.json` output: https://pastebin.com/5YMsvprs
- No new errors, just the warnings from #412

**Comment by kevinfodness** [comment]
I was able to reproduce this issue locally. There is definitely a bug in how we're processing Facebook embeds, which I believe is due to Facebook changing something about how their embeds work.

**Comment by kevinfodness** [comment]
Resolved via #461.
